package.json

```diff
@@ -61,16 +68,17 @@
     "mock-require": "^2.0.2",
     "rimraf": "^2.5.4",
     "sinon": "^2.3.1",
-    "ts-loader": "^2.1.0",
+    "ts-loader": "^4.0.0-beta.0",
     "tslint": "^5.0.0",
-    "typescript": "^2.1.0",
+    "typescript": "^2.6.2",
     "vue": "^2.5.9",
     "vue-class-component": "^6.1.1",
     "vue-loader": "^13.5.0",
     "vue-template-compiler": "^2.5.9",
-    "webpack": "^3.0.0"
+    "webpack": "^4.0.0-beta.0"
   },
   "peerDependencies": {
+    "tslint": "^5.0.0",
     "typescript": "^2.1.0",
     "webpack": "^2.3.0 || ^3.0.0"
```

Comment:
|| ^4.0.0? :)

Reply:
It will be backward compatible, right?

Reply:
Perhaps - I'm not sure to be honest.  Happy to add ^4.0.0 though.